### ISSUE TITLE
No Verify

### ISSUE THREAD

**Issue by procount** [raspberrypi/rpi-imager#72]
Would it be possible to add a `--noverify` option for the more advanced user? It would save some time when trying to flash images in succession, for those who know the risks.

**Comment by ghollingworth** [comment]
Probably would be best as a pull request (I'm unlikely to approve this functionality as a general addition, but pulling in a change and give it the once over sounds simple enough)

**Comment by tdewey-rpi** [comment]
Following up on this - does the skip option in the UI not cover this in modern Imager?

**Comment by procount** [comment]
I'm still using the old version. :wink:
But I looked at the documentation and it looks like it would do the trick.
Is the --cli option still available? If so, then, a '--skip' or '--noverify' option would be useful to set this feature for automating the image writing process in a script or whatever. 
Otherwise, you can close this as complete.

**Comment by tdewey-rpi** [comment]
Yes, `--cli` is still available in the GUI applications, and on Linux there's also the cli-only packages.

The equivalent of skipping verification for both would be to omit the `--sha256` argument, at which point Imager has no reference point and would skip verification.
